sklearn/decomposition/_pca.py

```diff
@@ -271,26 +273,30 @@ class PCA(_BasePCA):
 
     References
     ----------
-    For n_components == 'mle', this class uses the method of *Minka, T. P.
-    "Automatic choice of dimensionality for PCA". In NIPS, pp. 598-604*
+    For n_components == 'mle', this class uses the method from:
+    `Minka, T. P.. "Automatic choice of dimensionality for PCA".
+    In NIPS, pp. 598-604 <https://tminka.github.io/papers/pca/>`_
```

Comment:
Why not giving the same link as in the ``_assess_dimension` function (https://proceedings.neurips.cc/paper/2000/file/7503cfacd12053d309b6bed5c89de212-Paper.pdf) ? It links directly to a pdf.

Reply:
The link I give here links to Minka's page which has the pdf of the original publication, but also an updated and expanded version that explains the algorithm in more detail. I found it much easier to understand. People reading this general docstring may want to understand the algorthim in details. On the other hand, in the `_assess_dimension` there is an explicit reference to "equation 31", so one needs to look at the exact paper.

Reply:
Thanks for the clarification. Then I think it's nicer to the reader to link directly to the pdf of the expanded version (https://tminka.github.io/papers/pca/minka-pca.pdf)

Reply:
Done.